src/Microsoft.ML.Core/Data/ISchemaBindableMapper.cs

```diff
@@ -61,15 +66,20 @@ public interface ISchemaBoundRowMapper : ISchemaBoundMapper, IRowToRowMapper
     /// return a subset of the input columns.
     /// This interface is similar to <see cref="ISchemaBoundRowMapper"/>, except it does not have any input role mappings,
     /// so to rebind, the same input column names must be used.
-    /// Implementing of this object are typically created using a definie input <see cref="ISchema"/>.
+    /// Implementation of this interface are typically created using a defined input <see cref="ISchema"/>.
     /// </summary>
-    public interface IRowToRowMapper : ISchematized
+    public interface IRowToRowMapper
     {
         /// <summary>
         /// Mappers are defined as accepting inputs with this very specific schema.
         /// </summary>
         Schema InputSchema { get; }
 
+        /// <summary>
+        /// Gets an instance of <see cref="Schema"/> which describes the variables' names and types in the output generated by this mapper.
```

Comment:
>variables [](start = 73, length = 9)

Not really variables, columns, right?

Reply:
Yes, you're right.

---
In reply to: [237587946](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237587946) [](ancestors = 237587946)